test/providers/dynect_test.rb

```diff
@@ -204,8 +195,7 @@ def test_build_txt_from_api_handles_mixed_case_fqdn
       "record_type" => "TXT",
       "rdata" => {
         "txtdata" => "Hello, world!",
-      },
-    )
+      },)
```

Comment:
This feels weird to have the comma in 'sandwich' between closing chars.
If you fix the first line instead (as suggested above) this will be fine